components/dashboard/src/teams/Members.tsx

```diff
@@ -58,7 +57,7 @@ export default function () {
 
     const getInviteURL = (inviteId: string) => {
```

Comment:
definitely, but I'd like to avoid including even more changes in my PR. There are so many places where we don't properly manage the life cycle of react components.